FirebaseMessaging/Sources/FIRMessaging.m

```diff
@@ -143,14 +144,16 @@ + (FIRMessagingExtensionHelper *)extensionHelper {
 }
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
-- (instancetype)initWithAnalytics:(nullable id<FIRAnalyticsInterop>)analytics
-                 withUserDefaults:(GULUserDefaults *)defaults {
+- (instancetype)initForApp:(FIRApp *)app
+             withAnalytics:(nullable id<FIRAnalyticsInterop>)analytics
```

Comment:
`with` isn't necessary on the second argument if it's used for the first

Reply:
Reworked in a3dc31a